kedro/config/omegaconf_config.py

```diff
@@ -6,6 +6,7 @@
 import io
 import logging
 import mimetypes
+import os
```

Comment:
We actually try to not use `os` if it's possible.. Is there no other way to achieve this with using `pathlib`?